<launch>

  <arg name="debug" default="false"/>
  <arg if="$(arg debug)" name="launch_prefix" value="xterm -rv -e gdb -ex run -args"/>
  <arg unless="$(arg debug)" name="launch_prefix" value=""/>

  <node name="ARLeftMultipleSinglePattern" args="ARLeftMultipleSinglePattern" pkg="ar_target" type="ar_multiple_single" respawn="false" output="screen" launch-prefix="$(arg launch_prefix)">
    <rosparam command="load" file="$(find ar_target)/config/ar_multiple_single_marker.yaml"/>
    <param name="camera_frame" type="string" value="/BUMBLEBEE_LEFT"/>    
    <param name="marker_frame" type="string" value="ARLeftMarker"/>
    <param name="camera_image_topic" type="string" value="/Honeybee/left/image_rect_color"/>
    <param name="camera_info_topic" type="string" value="/Honeybee/left/camera_info"/>
    <param name="marker_red" type="double" value="0.0"/>
    <param name="marker_green" type="double" value="1.0"/>
    <param name="marker_blue" type="double" value="0.0"/>
  </node>
</launch>
